src/main/java/com/amazon/sqs/javamessaging/PrefetchManager.java

```diff
@@ -25,19 +25,19 @@ public interface PrefetchManager {
      * messageQueue when user calls for receive or message listener onMessage is
      * called.
      */
-    public void messageDispatched();
+    void messageDispatched();
```

Comment:
It is still better to retain the class decoration so to be conformed with the package convention